### PR TITLE
Test search engine

### PR DESCRIPTION
**Pull request by uyen3893**
Please review it

### CODE REVIEW

File: test/nop-page-objects/search.engine.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    }
+    get search_button() {
+        return $('//button[@type="submit"]')
+    }
+        
+    get list_of_products() {
+        return $$('//div[@class="item-box"]')
+    }   
+
+    // get first_title() {
```
Comment (phkhuong) [inline test/nop-page-objects/search.engine.js]: Please remove commented code

File: test/specs/test_search_engine.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+describe('Search engine', () => {
+    before(() => {
+        Search.open()
+        Search.setFrame()
+    })
+
+    describe('Input valid keyword', () => {
+        it('1. should show two apple\'s products', () => {
+            Search.search('apple')
+            console.log('uyen2: ' + JSON.stringify(Search.get_title_of_product(0)))
```
Comment (phkhuong) [inline test/specs/test_search_engine.js]: Please remove log

**Review by phkhuong — CHANGES_REQUESTED** [review]
Please fix as comments

File: test/nop-page-objects/search.engine.js (hunk 1 above)
Comment (uyen3893) [reply test/nop-page-objects/search.engine.js]: I fixed it

File: test/specs/test_search_engine.js (hunk 2 above)
Comment (uyen3893) [reply test/specs/test_search_engine.js]: I fixed it

**Review by phkhuong — APPROVED** [review]

**Pull request merged**